web/src/views/FileBrowserView/FileBrowser.vue

```diff
@@ -230,20 +244,40 @@ import { trimEnd } from 'lodash';
 
 import { dandiRest } from '@/rest';
 import { useDandisetStore } from '@/stores/dandiset';
-import { AssetFile, AssetFolder, AssetStats } from '@/types';
+import { AssetFile, AssetPath } from '@/types';
 
-const parentDirectory = '..';
 const rootDirectory = '';
-
 const FILES_PER_PAGE = 15;
 
-const sortByName = (a: AssetStats, b: AssetStats) => {
-  if (a.name > b.name) {
+// AssetService is slightly different from Service
+export interface AssetService {
```

Comment:
```suggestion
interface AssetService {
```

Does this need to be exported? I don't see it used anywhere outside of this component.